java/src/jmri/jmrit/logixng/tools/swing/LogixNGEditor.java

```diff
@@ -751,7 +751,8 @@ public final class ConditionalNGTableModel extends AbstractTableModel
 
         public static final int SNAME_COLUMN = 0;
         public static final int UNAME_COLUMN = SNAME_COLUMN + 1;
-        public static final int THREAD_COLUMN = UNAME_COLUMN + 1;
+        public static final int ENABLED_COLUMN = UNAME_COLUMN + 1;
```

Comment:
I think it is better to have enabled and startup together.

Reply:
I agree.

There will be a merge conflict between your PR and mine due to the changes to the release notes so I will fix this and merge upstream/master into this PR once your PR is merged.